MIGRATION.md

```diff
@@ -152,11 +152,11 @@ resource "stackit_logme_credential" "example-credential" {
 | stackit_observability_instance           | stackit_observability_instance          | :white_check_mark: | [project_id],[instance_id] |                                                                  |
 | stackit_observability_job                | stackit_observability_scrapeconfig      | :white_check_mark: | [project_id],[instance_id],[name] |                                                                  |
 | stackit_opensearch_credential            | stackit_opensearch_credential           | :white_check_mark: | [project_id],[credentials_group_id],[credential_id] |                                                                  |
-| stackit_opensearch_instance              | stackit_opensearch_instance             | :white_check_mark: | [project_id],[instance_id] |                                                                  |
-| stackit_postgres_credential              | stackit_postgresql_credential           | :white_check_mark: | [project_id],[credentials_group_id],[credential_id] |                                                                  |
+| stackit_opensearch_instance              | stackit_opensearch_instance             | :white_check_mark: | [project_id],[instance_id] |                                                                  |                                 |
 | stackit_postgres_flex_instance           | stackit_postgresflex_instance           | :white_check_mark: | [project_id],[instance_id] |                                                                  |
 | stackit_postgres_flex_user               | stackit_postgresflex_user               | :warning: | [project_id],[instance_id],[user_id] | `password` field will be empty                                   |
-| stackit_postgres_instance                | stackit_postgresql_instance             | :white_check_mark: | [project_id],[instance_id] |                                                                  |
+| stackit_postgres_instance                |                                          | | | Resource deprecated                                            |
```

Comment:
Should we remove these lines completely?

Reply:
It's similar to other deprecated resources like `elasticsearch`, we keep them in the table even though we don't have them in our provider